models/Issue.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var issueSchema = mongoose.schema({
+	author_id: Number,
```

Comment:
Did we decide to create an incrementing author_id or did we want to just us the _id that was generated by Mongo? I am still unclear about that.